packages/core/src/BaseNavigationContainer.tsx

```diff
@@ -199,6 +199,14 @@ const BaseNavigationContainer = React.forwardRef(
       return getStateForRoute('root');
     }, [getStateForRoute]);
 
+    const getCurrentRoute = React.useCallback(() => {
+      let state: NavigationState = getRootState() as NavigationState;
```

Comment:
The state could be `undefined` if there's no navigator rendered. We should probably add a check to avoid `undefined is not an object` error.